packages/backend/src/graphql/queries/tiles/get-all-rows.ts

```diff
@@ -0,0 +1,26 @@
+import { getAllTableRows, TableRowSchema } from '@/models/dynamodb/table-row'
+import Context from '@/types/express/context'
+
+type Params = {
+  input: {
+    tableId: string
+  }
+}
+
+const getAllRows = async (
```

Comment:
nit: would rather leave this as `getRows` and think about adding pagination to this later. I want to avoid other code relying on the ability to get _all_ rows.